resilience4j-framework-common/src/main/java/io/github/resilience4j/common/retry/configuration/CommonRetryConfigurationProperties.java

```diff
@@ -342,9 +341,9 @@ public Duration getWaitDuration() {
 
         public InstanceProperties setWaitDuration(Duration waitDuration) {
             Objects.requireNonNull(waitDuration);
-            if (waitDuration.toMillis() < 0) {
+            if (waitDuration.isNegative()) {
```

Comment:
Not sure what to do here: it allow 0 value [here](https://github.com/resilience4j/resilience4j/blob/a736c0e4f8224ac5d3195c748aea7ca6aab8a915/resilience4j-framework-common/src/main/java/io/github/resilience4j/common/retry/configuration/CommonRetryConfigurationProperties.java#L179) but prohibits [here](https://github.com/resilience4j/resilience4j/blob/b81dd07aaa10a8f69fa983ed03e14f062048c552/resilience4j-core/src/main/java/io/github/resilience4j/core/IntervalFunction.java#L247). I've chosen less restrictive change so values between 0 and 1 ms will fail during handling